service/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
 	implementation 'javax.xml.soap:javax.xml.soap-api:1.4.0'
 	implementation 'com.github.spullara.mustache.java:compiler:0.9.5'
 	implementation 'org.apache.tika:tika-core:1.26'
+	implementation 'org.awaitility:awaitility:4.0.3'
```

Comment:
Do we need this dependency? it is only used once in test and it has been already added as test dependency.